### PR TITLE
Add troubleshooting in local development

### PR DESCRIPTION
**Pull request by hillairet**
## What

Fix a couple of typos and add troubleshooting section in developing locally page to solve an issue I encountered.

## How

The Java home folder needs to be explicitly defined.
This is a documentation update so I haven't gone through the checklist below.

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

### CODE REVIEW

File: docs/contributing-to-airbyte/developing-locally.md [hunk 1]
```diff
@@ -156,3 +156,14 @@ Sometimes you'll want to reset the data in your local environment. One common ca
...
+## Troubleshooting
+
+### `gradlew Could not target platform: 'Java SE 14' using tool chain: 'JDK 8 (1.8)'.`
+
+Somehow gradle didn't pick up the right java version for some reason.
+Find the install version and set the `JAVA_HOME` environment to point to the JDK folder.
+
+For example:
+```
+env JAVA_HOME=/usr/lib/jvm/java-15-openjdk ./gradlew  :airbyte-integrations:connectors:your-connector-dir:build
```
Comment (sherifnada) [inline docs/contributing-to-airbyte/developing-locally.md:168]: Should this point to Java 14?

**Review by sherifnada — APPROVED** [review]
Thanks for the PR! One question but LGTM otherwise.

File: docs/contributing-to-airbyte/developing-locally.md (hunk 1 above)
Comment (sherifnada) [inline docs/contributing-to-airbyte/developing-locally.md:168]:
```suggestion
env JAVA_HOME=/usr/lib/jvm/java-14-openjdk ./gradlew  :airbyte-integrations:connectors:your-connector-dir:build
```

Comment (hillairet) [reply docs/contributing-to-airbyte/developing-locally.md:168]: Good call. I think I have java 15 on my machine and that's where I needed this command. It seems to work fine with 15.

**Pull request merged**